dts/bindings/misc/tenstorrent,bh-fwtable.yaml

```diff
@@ -9,8 +9,9 @@ include: base.yaml
 
 properties:
 
-  flash-dev:
+  flash_dev:
     type: phandle
+    required: true
```

Comment:
This should probably be reverted (dt properties should use dashes, and I think we want to allow not specifying the flash device (as a special means of indicating that we should skip load firmware tables).